fhirflat/ingest.py

```diff
@@ -458,6 +460,8 @@ def convert_data_to_flat(
         be named by resource, and contain the mapping for that resource.
     subject_id: str
         The name of the column containing the subject ID in the data file.
+    zip_folder: bool
```

Comment:
Can use `compress_format` and allow user to use any of the options in `shutil.make_archive`